app/assets/stylesheets/active_admin_sidebar.scss

```diff
@@ -0,0 +1,87 @@
+@import "active_admin_sidebar_pure_icons";
+
+body.active_admin {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `active_admin` should be written in lowercase with hyphens